bin/cloudflare_worker/Cargo.toml

```diff
@@ -13,7 +13,7 @@ wasm-opt = true
 crate-type = ["cdylib"]
 
 [dependencies]
-worker = { git = "https://github.com/dotansimha/workers-rs.git", rev = "6de1efc2ef155236d8caf376cce2a0f9972e241f" }
+worker = "0.4.2"
```

Comment:
I don't remember the reason why forking `worker` was necessary, but I see that switching to the latest has no compiler issues, so I guess it's fine? 🤔